pom.xml

```diff
@@ -94,20 +94,6 @@
                     <transformers>
                         <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
                     </transformers>
-                    <relocations>
```

Comment:
If you compile together with relocations then the plugin is going to but does not work, the issus is already created (https://github.com/graylog-labs/graylog-plugin-jabber/issues/11) but not solved